### DISCUSSION TITLE
[Request] Add Hyperspace to Brew

### DISCUSSION THREAD

**Discussion by iero** [hyperspacedev/hyperspace#135] [Ideas]
As Hyperspace is not available in Mac App Store, you could make it available to Brew:
https://github.com/Homebrew/homebrew-cask
https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md

[a comment by an automated account is not shown]

**Comment by alicerunsonfedora** [comment]
We'll definitely take a look into it for the Desktop version. We are planning to also make a new version of Hyperspace dedicated to macOS/iOS that should be in the App Store at some point.

Noted as [HD-26]

[HD-26]: https://hyperspacedev.atlassian.net/browse/HD-26
